config/opal_setup_cc.m4

```diff
@@ -41,14 +154,7 @@ AC_DEFUN([OPAL_SETUP_CC],[
     WRAPPER_CC="$CC"
     AC_SUBST([WRAPPER_CC])
 
-    # From Open MPI 1.7 on we require a C99 compiant compiler
-    AC_PROG_CC_C99
-    # The result of AC_PROG_CC_C99 is stored in ac_cv_prog_cc_c99
-    if test "x$ac_cv_prog_cc_c99" = xno ; then
-        AC_MSG_WARN([Open MPI requires a C99 compiler])
-        AC_MSG_ERROR([Aborting.])
-    fi
-
```

Comment:
I kinda like the C99 check out here, independent of the C11 check.

Yes, it's (minorly) redundant with the C11 checks -- but the code is very clear this way.

Also -- it might be worth updating the `AC_MSG_WARN` message to say something like "Open MPI requires a C99 (or later) compiler".

Reply:
Ok, moving it now.